CommunityFridgeMapApi/tests/unit/test_tag.py

```diff
@@ -0,0 +1,57 @@
+from multiprocessing import connection
+
+from dependencies.python.db import layer_test
+from dependencies.python.db import get_ddb_connection
+import unittest
+from dependencies.python.db import Tag
+
+def test_layer_test():
+
+    ret = layer_test()
+    assert ret == "hello world"
+
+def test_get_ddb_connection():
```

Comment:
You don't have to test this function because it's already tested in the Fridge test file I believe